app/grandchallenge/core/static/css/core.css

```diff
@@ -218,3 +218,15 @@ a[aria-expanded=true] .fa-chevron-right {
 a[aria-expanded=false] .fa-chevron-down {
    display: none;
 }
+
+
+/* htmx */
+.htmx-indicator {
+ display:none;
+}
+.htmx-request .htmx-indicator {
+    display:inline-block;
+}
+.htmx-request.htmx-indicator {
+    display:inline-block;
+}
```

Comment:
Why do you need special class definitions for this? Can't you use `d-block` and `d-inline-block`?

Reply:
We need to use special classes to select the correct element at the correct time. We could use d-none instead of `htmx-indicator` but that seems hacky.

Reference: https://htmx.org/attributes/hx-indicator/